MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,72 +1,169 @@
+                    self?.present(alert, animated: true, completion: nil)
+                }
+    }
+    
+    @IBAction private func noButtonClicked(_ sender: UIButton) {
+        guard let currentQuestion = currentQuestion else {
+            return
+        }
+        
+        showAnswerResult(isCorrect: currentQuestion.correctAnswer == false)
+    }
+    @IBAction private func yesButtonClicked(_ sender: UIButton) {
+        guard let currentQuestion = currentQuestion else {
+            return
+        }
+        
+        showAnswerResult(isCorrect: currentQuestion.correctAnswer == true)
+    }
+    @IBOutlet weak private var imageView: UIImageView!
+    @IBOutlet weak private var textLabel: UILabel!
+    @IBOutlet weak private var counterLabel: UILabel!
+    
+    @IBOutlet weak private var noUIButton: UIButton!
+    
+    @IBOutlet weak private var yesUIButton: UIButton!
+   
+    private var correctAnswers: Int = 0
+    private var currentQuestionIndex: Int = 0
+    
+    private let questionsAmount: Int = 10
+    private var questionFactory: QuestionFactoryProtocol?
+    private var currentQuestion: QuizQuestion?
+    
+    private var alertPresenter: AlertPresenter?
+    private var statisticService: StatisticService?
+    
+    private func show(quiz step: QuizStepViewModel) {
+        imageView.layer.borderWidth = 0
+        imageView.image = step.image
+        textLabel.text = step.question
+        counterLabel.text = step.questionNumber
+        
+        turnOnButtons()
     }
-}
-
-/*
- Mock-данные
- 
- 
- Картинка: The Godfather
- Настоящий рейтинг: 9,2
- Вопрос: Рейтинг этого фильма больше чем 6?
- Ответ: ДА
-
-
- Картинка: The Dark Knight
- Настоящий рейтинг: 9
- Вопрос: Рейтинг этого фильма больше чем 6?
- Ответ: ДА
-
-
- Картинка: Kill Bill
- Настоящий рейтинг: 8,1
- Вопрос: Рейтинг этого фильма больше чем 6?
- Ответ: ДА
-
-
- Картинка: The Avengers
- Настоящий рейтинг: 8
- Вопрос: Рейтинг этого фильма больше чем 6?
- Ответ: ДА
-
-
- Картинка: Deadpool
- Настоящий рейтинг: 8
- Вопрос: Рейтинг этого фильма больше чем 6?
- Ответ: ДА
-
 
- Картинка: The Green Knight
- Настоящий рейтинг: 6,6
- Вопрос: Рейтинг этого фильма больше чем 6?
- Ответ: ДА
+    private func show(quiz result: QuizResultsViewModel) {
+
+        guard let statisticService = statisticService else { return }
+        
+        let alertModel = AlertModel(
+            title: result.title,
+            message: """
+                Ваш результат: \(statisticService.currentGame.correct)/\(statisticService.currentGame.total)\n
+                Количество сыгранных квизов: \(statisticService.gamesCount)\n
+                Рекорд: \(statisticService.bestGame.correct)/\(statisticService.bestGame.total) (\(statisticService.bestGame.date.dateTimeString))\n
+                Средняя точность: \(String(format: "%.2f", statisticService.totalAccuracy*100))%
+                """,
+            buttonText: result.buttonText,
+            completion: {[weak self] in
```

Comment:
После фигурной либо пробел либо ентер
```suggestion
            completion: { [weak self] in
```